runestone/activecode/js/MD5.js

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
OK, so my last comment is simply that if you copied this from somewhere else that we can publicly link to, lets just do that.  There's no reason to have a copy that can get out of date when someone else fixes a bug in the code you copied.